.mci.yml

```diff
@@ -3,6 +3,17 @@
 #######################################
 
 #######################################
+#            Variables                #
+#######################################
+
+linux_cmake_path: &linux_cmake_path /opt/cmake/bin/cmake
```

Comment:
Looking at this again maybe it makes more sense to organize these by feature?  Up to you!